cmd/zetacored/root.go

```diff
@@ -268,48 +268,44 @@ func (ac appCreator) newApp(
 	)
 }
 
-// appExport creates a new simapp (optionally at a given height)
+// appExport is used to export the state of the application for a genesis file.
 func (ac appCreator) appExport(
 	logger log.Logger, db dbm.DB, traceStore io.Writer, height int64, forZeroHeight bool, jailAllowedAddrs []string,
 	appOpts servertypes.AppOptions,
 	modulesToExport []string,
 ) (servertypes.ExportedApp, error) {
-	var anApp *app.App
+	var zetaApp *app.App
 
 	homePath, ok := appOpts.Get(flags.FlagHome).(string)
 	if !ok || homePath == "" {
 		return servertypes.ExportedApp{}, errors.New("application home not set")
 	}
 
+	loadLatest := false
```

Comment:
Why this change in this file? looks unrelated to the purpose of the PR?

Reply:
It does not change the logic , just restructures the code to make it more readable .
I just came across this when investigating , the change seems too small to be part of a separate pr . lmk if you think otherwise